pdfmaker/css/torDOTcom/novel.scss

```diff
@@ -56,6 +57,8 @@ $sizefootnote: 9pt;
 $sizecaption: $smaller;
 $sizefigsrc: 7pt;
 $sizesuperscript: 6pt;
+$sizehwritingprint: $gridheight * .8;
+$sizehwritingnormal: $gridheight * .69;
```

Comment:
I don't think it's necessary to add these font sizes as a function of grid height. It's better to match Jamie's font size specs exactly, by just hardcoding it into this variable. You already need to define the variable in two different places (novella.scss and novel.scss), so you might as well give Jamie exactly what he wants. Font size isn't something that needs to always relate to grid height--it's a more flexible design relationship.